.travis.yml

```diff
@@ -28,10 +30,11 @@ install:
   - ./travis-tool.sh r_binary_install servr
   - ./travis-tool.sh r_binary_install XML
   - ./travis-tool.sh r_binary_install shiny
+  - ./travis-tool.sh r_binary_install RSelenium
```

Comment:
same thing, shouldn't we delete this if we install_github ropensci/RSelenium later?